server/testdriver/junit4/pom.xml

```diff
@@ -49,6 +48,16 @@
          <artifactId>shrinkwrap-resolver-impl-maven-archive</artifactId>
          <scope>compile</scope>
       </dependency>
+      <dependency>
+         <groupId>junit</groupId>
+         <artifactId>junit</artifactId>
+         <scope>compile</scope>
+      </dependency>
+      <dependency>
+         <groupId>org.testng</groupId>
+         <artifactId>testng</artifactId>
+         <scope>test</scope>
```

Comment:
Do we really need TestNG here? Maybe add a source comment to explain why

Reply:
checking